src/stories/Sidebar/Sidebar.mdx

```diff
@@ -0,0 +1,194 @@
+import { Primary, ArgsTable, Canvas, Story } from "@storybook/addon-docs";
```

Comment:
Let's add an example of a highly customized sidebar that can be used as a template for an app to reference